frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/aoeQuestionComponents/GenderOfSexualPartnersAoe.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
```

Comment:
Yay! Thank you for making these components! 🎉 If we have the bandwidth it could be nice to clean the other usages up to leverage these components!

Reply:
Planning on making a follow up PR so the other AOE forms can use these components too. Trying to componentize the full symptom flow of multiple questions could be a little challenging though 🤔